Cargo.toml

```diff
@@ -63,6 +64,15 @@ version = "1.4"
 default-features = false
 optional = true
 
+[dependencies.serde]
+version = "1.0.197"
+default-features = false
+optional = true
+
+[dependencies.serdect]
+version = "0.2.0"
```

Comment:
One important caveat on `serdect` is we [revised the format in the v0.3.0-pre.0 prereleases](https://github.com/RustCrypto/formats/pull/1112), using `serialize_bytes`/`deserialize_bytes` which unfortunately adds a length header in the binary encodings even when using fixed-sized arrays but provides more compact serialization on formats like MessagePack which are more likely to be constant-time-ish, which is an unfortunate tradeoff where we opted for broader constant time assuredness over a minimally compact encoding.

Barring any complaints about the length header, we just need to fix a bug/regression on v0.3.0 in the slice encoder and we can ship a final release. I would probably suggest using that (or convincing us to roll back) to avoid future breakages in the encoding format.

Reply:
When would you expect to ship v0.3.0?

Reply:
This regression needs to be fixed first: https://github.com/RustCrypto/formats/issues/1322